### DISCUSSION TITLE
SIP configuration

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#467] [Ideas] [answered]
how does that work and how can the function be integrated?

**Comment by szaimen** [comment]
Honestly this is rather a feature request or question so moving to discussions.

**Comment by szaimen** [comment, marked as answer]
As far as I know is no documentation on this available so nothing that we can answer or implement here.

**Reply by ghost** [reply]
But if there is no information about this, why does the SIP setting in Talk exist at all?

**Reply by szaimen** [reply]
I suppose because of https://nextcloud.com/talk/#:~:text=sip%20bridge%3A%20dial%20in%20by%20phone2

**Reply by ghost** [reply]
Because of what? I can't get the answer from the entry.

**Reply by szaimen** [reply]
Then try to open that link with a browser that understands this kind of links like Google Chrome or Microsoft Edge.

**Reply by ghost** [reply]
yes that worked.

But that should be self-hosted still intigratable!?

https://github.com/nextcloud/spreed/issues/5370
https://github.com/nextcloud/spreed/issues/5370#issuecomment-823379889

**Reply by szaimen** [reply]
Not sure if you read the whole thread and I repeat myself but there is no documentation available on SIP integration as you can also get from these threads.

**Reply by Fuseteam** [reply]
this is the most i can find now https://help.nextcloud.com/t/sip-gateway-setup/118109/5
